### PR TITLE
7 implement embeddings method

### LINKED ISSUE #7: Implement embeddings method
We need to add a method to create embeddings that the Mistral AI provides.

To get started, you can check the [official API](https://docs.mistral.ai/api/#operation/createEmbedding).

### PR DESCRIPTION
**Pull request by MarcinGruchala**
Resolves [issue#7](https://github.com/nomtek/mistralai_client_dart/issues/7) 
 
- Fix the typo in a method name
- Add embeddings method implementation 
- Add `EmbeddingsParams` model docs

### CODE REVIEW

File: lib/src/mistralai_client_dart_base.dart [hunk 1]
```diff
@@ -62,7 +63,37 @@ class MistralAIClient {
     throw UnimplementedError('to be implemented');
   }
 
-  Future<Embeddings> embednings(EmbeddingParams params) {
-    throw UnimplementedError('to be implemented');
+  /// Returns embeddings [Embeddings] for a single,
+  /// or batch of inputs given as params [EmbeddingParams]
```
Comment (KacperKluka) [inline lib/src/mistralai_client_dart_base.dart]:
```suggestion
  /// Returns [Embeddings] for a single input
  /// or a batch of inputs given as [EmbeddingParams]
```

**Review by KacperKluka — CHANGES_REQUESTED** [review]
Just a small fix in the docs, overall good job 👍

**Review by KacperKluka — APPROVED** [review]
LGTM 👍

**Review by gawi151 — APPROVED** [review]
👍

**Pull request merged**